blockly-ws-search/index.html

```diff
@@ -58,7 +58,6 @@
   var ws = Blockly.inject('blocklyDiv' + suffix, options);
   var workspaceSearch = new WorkspaceSearch(ws);
   workspaceSearch.init();
-  workspaceSearch.open();
```

Comment:
I think it would be nice to open on page load, just for convenience of our own testing.